spec/models/page_spec.rb

```diff
@@ -61,6 +61,31 @@
         expect(page).to be_invalid
         expect(page.errors[:page_heading]).to include("must be present when Guidance Markdown is present")
       end
+
+      context "when markdown is too long" do
+        it "adds an error to guidance_markdown" do
+          page.guidance_markdown = "ABC" * 5000
+          expect(page).to be_invalid
+          expect(page.errors[:guidance_markdown]).to include("is too long (maximum is 4999 characters)")
+        end
```

Comment:
It's tempting to suggest that the length limit should be something that's defined within the gem, and we can draw from it if we want to pick out the limit - might make a card for it if we end up having to change the limit a bunch

Reply:
Yeah, I agree. Lets keep it simple for the moment until we do other validation/customisations